pkgs/top-level/all-packages.nix

```diff
@@ -2534,9 +2534,7 @@ in
 
   monetdb = callPackage ../servers/sql/monetdb { };
 
-  monado = callPackage ../applications/graphics/monado {
-    inherit (gst_all_1) gstreamer gst-plugins-base;
-  };
+  monado = gst_all_1.callPackage ../applications/graphics/monado {  };
```

Comment:
I think we should avoid this pattern in package sets: using the `callPackage` from a different scope breaks composition. In these cases it is in my opinion much better to pass in the subpackage set.

Reply:
Related RFC https://github.com/NixOS/rfcs/pull/83.